zebra-consensus/src/config.rs

```diff
@@ -12,6 +12,8 @@ pub struct Config {
     /// Future versions of Zebra may change the mandatory checkpoint
     /// height.
     pub checkpoint_sync: bool,
+    /// Skip the download of Groth16 parameters if this option is true.
```

Comment:
Nit: The parameters are always downloaded if they are used.

This option only skips the pre-download that happens as soon as Zebra is launched. (Rather than waiting until we start full verification to do the download.)

```suggestion
    /// Skip the pre-download of Groth16 parameters if this option is true.
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/3197/commits/21015e9d36e5ee3394577dd77faa5d10f837b22f